apps/kyb-app/src/pages/CollectionFlow/versions/v2/CollectionFlowV2.tsx

```
@@ -164,15 +175,22 @@ export const CollectionFlowV2 = withSessionProtected(() => {
                                       ) : null}
                                     </div>
                                     <div>
-                                      {customer?.displayName && (
+                                      {themeDefinition.settings.contactInformation ? (
+                                        <div
+                                          className="text-sm"
+                                          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)